app/src/main/java/org/mozilla/fenix/home/recenttabs/RecentSyncedTabHomeFeature.kt

```diff
@@ -0,0 +1,124 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.recenttabs
```

Comment:
Maybe move all of this into its own /home/recentsyncedtab/ package since we're breaking it out

Reply:
I didn't at first because I wasn't sure where to put shared components between the two packages. I think it makes sense to continue sharing some of the composables in `RecentTabs`, since it seems likely that the design for the two cards will stay largely in sync. I can move the recent synced tab stuff into its own package and just import the recent tabs composables for now, but IMO that shows they are candidates for a more generic UI package in the future.

Reply:
Can you make a list of the reusable composable components? The one obvious one I saw in my brief scan is `RecentTabTitle`, which I think we should probably refactor and use our existing reusable `PrimaryText()` component and define a typography. This can be a filed as a followup along with the list of things you see that we can refactor. Alternatively, I would refactor them in a separate issue and land it before this PR.